static/styles/settings.css

```diff
@@ -1797,6 +1797,12 @@ $option_title_width: 180px;
         }
 
         .sidebar {
+            position: absolute;
+            width: 100%;
+            border: none;
+            /* 48px is the height of settings header and 45px is the height of tab-container */
+            height: calc(100% - 93px);
```

Comment:
Can we use this variable at least? I don't see the 48px used directly in this file, so that may be harder to use as a variable, but generally we should be aiming to have any calculations like this using variables, since that makes it far easier to change things without introducing bugs.

```
/* The height of the settings header (including tab switcher). */                                        
$settings_header_height: 45px;                                                                           
```

Reply:
I have not used the variable as the variable is for header but the header height is actually 48px and 45px is of the tab container below it and this van cause confusion.

Reply:
OK, we can try to clean this up later I guess.